frontend/src/routes/manage/Video/index.tsx

```diff
@@ -99,7 +99,7 @@ const ManageVideos: React.FC<Props> = ({ connection, vars }) => {
     } else {
         inner = <>
             <PageNavigation {...{ vars, connection }} />
-            <div css={{ flex: "1 0 0" }}>
+            <div css={{ flex: "1 0 0", margin: "16px 0" }}>
```

Comment:
I made exactly the same changes to this file in #778 🤨

Reply:
Interesting how this doesn't result in a merge conflict or something ...